database.sql

```diff
@@ -169,7 +169,7 @@ ALTER TABLE `Student_Reviews`
 -- Contenu de la table `Discipline`
 --
 
-INSERT INTO 'Discipline' ('id', 'icon', 'name', 'description') VALUES
+INSERT INTO `Discipline` (`id`, 'icon', 'name', 'description') VALUES
```

Comment:
you don't need to insert id values since it is in auto increment as a primary key